ambry-router/src/test/java/com.github.ambry.router/CryptoJobHandlerTest.java

```diff
@@ -69,7 +69,7 @@ public CryptoJobHandlerTest() throws IOException, GeneralSecurityException {
     cryptoService = new GCMCryptoServiceFactory(verifiableProperties, REGISTRY).getCryptoService();
     cryptoJobHandler = new CryptoJobHandler(DEFAULT_THREAD_COUNT);
     referenceClusterMap = new MockClusterMap();
-    routerMetrics = new NonBlockingRouterMetrics(referenceClusterMap);
+    routerMetrics = new NonBlockingRouterMetrics(referenceClusterMap, null);
```

Comment:
I purposely set  `RouterConfig`  = `null` when creating `NonBlockingRouterMetrics` in several tests. This is to verify that even if router config is mistakenly set to null in the ctor of `NonBlockingRouterMetrics`, there would be no failure during instantiation.